int32 tag
float64 dist
geometry_msgs/PoseStamped pose
